src/findcdn/cdnEngine/detectCDN/cdn_check.py

```diff
@@ -116,7 +152,7 @@ def https_lookup(
     ) -> int:
         """Read 'server' header for CDN hints."""
         # List of domains with different protocols to check.
-        PROTOCOLS = ["https://", "https://www."]
```

Comment:
I think keeping the "www." here is bad so I am fine with removing it here and keeping with your change of actually having protocols instead bar the above comment keeping www.